src/core/core_structs/results_model.jl

```diff
@@ -0,0 +1,85 @@
+abstract type Results end
+
+get_results(result::Results) = nothing
+struct OperationsProblemResults <: Results
+    variables::Dict{Symbol, DataFrames.DataFrame}
+    total_cost::Dict
+    optimizer_log::Dict
+    time_stamp::DataFrames.DataFrame
+end
+
+get_duals(result::OperationsProblemResults) = nothing
+
+function make_results(variables::Dict{Symbol, DataFrames.DataFrame},
+                      total_cost::Dict,
+                      optimizer_log::Dict,
+                      time_stamp::DataFrames.DataFrame)
+    return OperationsProblemResults(variables, total_cost, optimizer_log, time_stamp)
+end
+
+function make_results(variables::Dict{Symbol, DataFrames.DataFrame},
+                      total_cost::Dict,
+                      optimizer_log::Dict,
+                      time_stamp::DataFrames.DataFrame,
+                      duals::Dict{Symbol, Any})
+    return AggregatedResults(variables, total_cost, optimizer_log, time_stamp, duals)
+end
+function get_variable(res_model::OperationsProblemResults, key::Symbol)
+        try
+            !isnothing(res_model.variables)
+        catch
+            error("No variable with key $(key) has been found.")
+        end
+    return get(res_model.variables, key, nothing)
```

Comment:
I think you're missing some updates from Jose.  This is a bug that he fixed today.